src/Shared/ParameterBindingMethodCache.cs

```diff
@@ -47,7 +47,7 @@ public bool HasTryParseMethod(ParameterInfo parameter)
         }
 
         public bool HasBindAsyncMethod(ParameterInfo parameter) =>
-            FindBindAsyncMethod(parameter) is not null;
+            FindBindAsyncMethod(parameter).Item1 is not null;
```

Comment:
I dislike the use of Item1 here
```suggestion
            var (expression, _) = FindBindAsyncMethod(parameter);
            return expression is not null;
```